### PR TITLE
Add aws direct

### PR DESCRIPTION
**Pull request by tgadiev**

### CODE REVIEW

File: {{cookiecutter.role_name}}/molecule/resources/provisioning/WINRM/destroy.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (vutkin) [inline {{cookiecutter.role_name}}/molecule/resources/provisioning/WINRM/destroy.yml]: WINRM/destroy.yml will be same as EPC/destroy.yml.
Will weagree with duplication?

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    security_group_name:
+      - default
+    wait_timeout: 1800
+    ssh_user: centos
+    groups:
+      - rhel_family
+
+  - name: test-aws-centos6-{{ cookiecutter.role_name }}
+    image: ami-1585c46a
+    instance_type: m4.large
```
[the remark this answers is not in the document]
Comment (tgadiev) [reply {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: centos6 doesn't run on m5

File: {{cookiecutter.role_name}}/molecule/resources/provisioning/WINRM/destroy.yml (hunk 1 above)
Comment (tgadiev) [reply {{cookiecutter.role_name}}/molecule/resources/provisioning/WINRM/destroy.yml]: Agree as discussed

File: {{cookiecutter.role_name}}/.gitlab-ci.yml [hunk 3]
```diff
@@ -21,23 +21,23 @@ Ansible lint:
...
   script:
     - molecule test -s default
   tags:
-    - docker
+    - aws
 
 AWS {{ cookiecutter.role_name }}:
   variables:
-    EPC_REGION: AWS-EUCENTRAL
+    AWS_REGION: us-east-1
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/.gitlab-ci.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/LICENSE [hunk 4]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019 EPAM Systems
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/LICENSE]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+driver:
+  name: ec2
+lint:
+  name: {{ cookiecutter.lint_name }}
+  options:
+    config-file: .yamllint
+platforms:
+  - name: test-aws-centos7-{{ cookiecutter.role_name }}
+    image: ami-9887c6e7
+    instance_type: m5.large
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+    role-file: requirements.yml
+driver:
+  name: ec2
+lint:
+  name: {{ cookiecutter.lint_name }}
+  options:
+    config-file: .yamllint
+platforms:
+  - name: test-aws-centos7-{{ cookiecutter.role_name }}
+    image: ami-9887c6e7
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+    instance_type: m5.large
+    region: us-east-1
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml (hunk 2 above)
Comment (kharkevich) [inline {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+    spot_price: 0.04
+    security_group_name:
+      - default
+    wait_timeout: 1800
+    ssh_user: centos
+    groups:
+      - rhel_family
+
+  - name: test-aws-centos6-{{ cookiecutter.role_name }}
+    image: ami-1585c46a
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+    instance_type: m4.large
+    region: us-east-1
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+    spot_price: 0.04
+    security_group_name:
+      - default
+    wait_timeout: 1800
+    ssh_user: centos
+    groups:
+     - rhel_family
+
+  - name: test-aws-ubuntu16-{{ cookiecutter.role_name }}
+    image: ami-09677e0a6b14905b0
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+    security_group_name:
+      - default
+    spot_price: 0.04
+    wait_timeout: 1800
+    ssh_user: ubuntu
+    groups:
+      - debian_family
+
+  - name: test-aws-ubuntu18-{{ cookiecutter.role_name }}
+    image: ami-012fd5eb46f56731f
```
Comment (kharkevich) [inline {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: should be replaced for using variables value from cookiecutter

File: {{cookiecutter.role_name}}/.gitlab-ci.yml (hunk 3 above)
Comment (tgadiev) [reply {{cookiecutter.role_name}}/.gitlab-ci.yml]: Done

File: {{cookiecutter.role_name}}/LICENSE (hunk 4 above)
Comment (tgadiev) [reply {{cookiecutter.role_name}}/LICENSE]: Done

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml (hunk 9 above)
Comment (tgadiev) [reply {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: Done

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml (hunk 10 above)
Comment (tgadiev) [reply {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: Done

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml (hunk 11 above)
Comment (tgadiev) [reply {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: Done

File: {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml (hunk 2 above)
Comment (kharkevich) [reply {{cookiecutter.role_name}}/molecule/cloud-aws-direct/molecule.yml]: ok

**Pull request merged**